atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractAtlasDbKeyValueServiceTest.java

```diff
@@ -660,21 +661,104 @@ public void testGetRangeOfTimestampsThrowsOnError() {
 
     @Test
     public void testCannotModifyValuesAfterWrite() {
+        Cell cell = Cell.create(row0, column0);
         byte[] data = new byte[1];
-        byte[] unmodifiedData = Arrays.copyOf(data, data.length);
+        byte[] unmodifiedData = writeToCell(cell, data);
```

Comment:
Not sure I understand - are you suggesting this to to (1) make sure `writeToCell` is doing the right thing, or (2) use `not(sameInstance(data))` to verify we get back different objects instead of the current modify-and-verify?